recipes/pds4_tools/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set name = "pds4_tools" %}
+{% set version = "1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pds4_tools-{{ version }}.zip
+  sha256: 8e6fdd0a9c4063c9a337b6ed3c52686da4f96bfba2a30e42ec57a6cf25fff459
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=2.6,>=3.3
```

```suggestion
    - python >=3.3
```
These requirements are redundant. Please fix in both places.

---

What is the suggestion `python >=3.3` as opposed to `python >=2.6`?

---

That's a good question. My reasons are that we don't publish new packages for python=2.7 on conda-forge anymore, so we have no way of testing if this package is actually compatible with python=2.7? I'm fine with accepting the minimum version as 2.6.